src/modules/display.js

```diff
@@ -0,0 +1,74 @@
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Vgufs5n18vzBAjzxI4Io/scores';
+
+const scoreDisp = () => document.querySelector('#list-scores');
```

Comment:
**_[OPTIONAL]_** it looks like `scoreDisp` and `scoresDisplay` functions namings are similar, 
Let's try to choose names that are easier for your code reader to understand 
without looking at your code 😉👉

Reply:
Try to think about what actually these variables represent. 👍

Reply:
### @od-c0d3r  Ooh I didn't pay attention to that, thank you for saying :)
### I'll do it right now!